server/index.ts

```diff
@@ -0,0 +1,20 @@
+import './bootstrap-globals';
+import { createExpressHandler } from './createExpressHandler';
+import express from 'express';
+import path from 'path';
+import { ServerlessFunction } from './types';
+
+const PORT = process.env.PORT ?? 8081;
+
+const app = express();
+app.use(express.json());
+
```

Comment:
Can we also add CORS headers?
```
app.use((req, res, next) => {
  res.header('Access-Control-Allow-Origin', '*');
  res.header('Access-Control-Allow-Headers', '*');
  next();
});
```

Reply:
Sure - just curious though, why would we want to add them? I'm planning on hosting the front-end assets from the same origin as this server, so I don't think we would need them.

Reply:
I was thinking for dev purposes only. I usually have different frontend accessing different servers locally. This is one of them :)
But it's ok not to add them if we're going to deploy this.

Reply:
I see. When the app is run in dev mode with `npm start`, the api requests made to `/token` will be [proxied](https://github.com/twilio/twilio-video-app-react/blob/master/package.json#L104) to `localhost:8081/token`. So from that app's point of view, everything is on the same origin in dev mode. So I don't think we'll need them for that. But if we ever need them in the future I'm happy to add them in.